src/registrar/tests/test_admin.py

```diff
@@ -1163,10 +1371,11 @@ def test_alphabetically_sorted_fk_fields_domain_application(self):
             )
 
     def test_alphabetically_sorted_fk_fields_domain_information(self):
+        self.maxDiff = None
         tested_fields = [
             DomainInformation.authorizing_official.field,
             DomainInformation.submitter.field,
-            # DomainInformation.creator.field,
+            #DomainInformation.creator.field,
```

Comment:
This particular field is failing due to a bug in the underlying class in common. There is another ticket to reduce some engineering complexity in common.py, so that is better suited for there